lib/util.js

```diff
@@ -314,6 +315,28 @@ function formatValue(ctx, value, recurseTimes) {
       keys.unshift('size');
     empty = value.size === 0;
     formatter = formatMap;
+  } else if (value instanceof ArrayBuffer) {
```

Comment:
`Object.keys((new ArrayBuffer(5))).length === 0`. Should this be placed in the shortcut path above?